app/views/devise/passwords/edit.mobile.haml

```diff
@@ -23,7 +23,7 @@
               = f.password_field :password_confirmation
 
           .controls
-            = f.submit t('devise.passwords.edit.change_password'), :class: 'btn btn-primary'
+            = f.submit t('devise.passwords.edit.change_password'), class: 'btn btn-primary'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.